app/ui-message/client/message.js

```diff
@@ -445,13 +445,7 @@ const findParentMessage = (() => {
 				},
 			}, { multi: true });
 			if (!Messages.findOne({ _id })) {
-				/**
-				 * Delete rid from message to not render it and to not be considred in last message
-				 * find from load history method what was preveting the load of some messages in
-				 * between the reals last loaded message and this one if this one is older than
-				 * the real last loaded message.
-				 */
-				delete msg.rid;
+				msg._hidden = true;
```

Comment:
Where did you ignored hidden messages when getting the latest message for the room?

Reply:
now I did I think